src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -485,7 +484,7 @@ public String getScript() {
     }
 
     public Map<String,String> getLoadedScripts() {
-        return ImmutableMap.copyOf(loadedScripts);
+        return Collections.unmodifiableMap(loadedScripts);
```

Comment:
No, this is not a replacement. Perhaps you meant
```suggestion
        return Collections.unmodifiableMap(new HashMap<>(loadedScripts));
```

?

Reply:
Fixed - thanks!